benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_hammerdb_logs/analyze_hammerdb_logs.py

```diff
@@ -103,3 +111,102 @@ def display_hammerdb_result(self, hammerdb_result: dict, workload_log_file: str)
         # Display the plot in the notebook
         output_notebook()
         show(p)
+
+    @staticmethod
+    def compare_results(result1: dict, result2: dict, legend_label1: str, legend_label2: str, database: str):
```

Comment:
I saw this method is repeated in other classes with minor changes. Can you write it in another class and extend it in this class?

Reply:
@athiruma, please see may latest fix using Object-oriented inheritance by adding analyze_workloads_operations.py class